src/spec-configuration/containerFeaturesOCI.ts

```diff
@@ -292,4 +297,43 @@ export async function fetchRegistryAuthToken(output: Log, registry: string, ociR
 		return undefined;
 	}
 	return token;
-}
\ No newline at end of file
+}
+
+// Lists published versions of a feature
+export async function getPublishedVersions(featureRef: OCIFeatureRef, output: Log): Promise<string[] | undefined> {
```

Comment:
nit, if you could add a comment with a link to the OCI distribution spec that would be nice for future us.